Add initialSelectedIndex property to Tabs

WRidder opened the pull request:
Fixes https://github.com/callemall/material-ui/issues/375. Also checks whether the index is not out of bounds w.r.t. the amount of tab children.

hai-cea commented:
@WRidder Can you please also add PropType validation as well?
```
initialSelectedIndex: React.PropTypes.number
```

Also, please add the prop to the docs here:
https://github.com/callemall/material-ui/blob/master/docs/src/app/components/pages/components/tabs.jsx#L58

Thanks!

WRidder commented:
Alright, updated proptype validation. I also added the proptype validation for tabWidth since it was specified in the docs already. Added initialSelectedIndex to the docs as well.

hai-cea commented:
Thanks @WRidder :+1:

ferrannp commented:
Uf thanks for this one, very convenient specially for routing when sending a direct link to somebody. Any idea when this will be released?

mmrtnz commented:
Hopefully we can get v0.7.2 ironed out by Wednesday at the latest.
